sos/plugins/origin.py

```diff
@@ -150,6 +150,7 @@ def setup(self):
             self.add_copy_spec([
                 self.node_cfg,
                 os.path.join(self.node_base_dir, "*.crt"),
+                os.path.join(self.node_base_dir, "*.conf"),
```

Comment:
Being a bit picky: the use of wildcards is usually discouraged, and in this case maybe we could reference the files with their full names? `{resolv,node-dnsmasq}.conf`

The existing `"*.crt"` wildcard is motivated by the fact that the cert file names are not fixed: they depend on the node's respective k8s object name.

Should we reference the files by full name instead?